<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="funkcja_celu_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Min of function</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="funkcja_celu_box">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="ilosc_zmiennych_label">
                <property name="text">
                 <string>Nr of variables</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="ilosc_zmiennych_spinbox">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>5</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="punkt_startowy_label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Starting point</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="x_10">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="x_20">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="x_30">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="x_40">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="x_50">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="ograniczenia_label">
          <property name="text">
           <string>Constrains:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QListWidget" name="lista_ograniczen">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;=0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="ograniczenie"/>
          </item>
          <item>
           <widget class="QPushButton" name="dodaj_ograniczenie">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="usun_ograniczenie">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="epsilon0_label">
                <property name="text">
                 <string>Episilon0 = 10^</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="epsilon0_spinbox">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>-12</number>
                </property>
                <property name="maximum">
                 <number>-3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="epsilon1_label">
                <property name="text">
                 <string>Epsilon1 = 10^</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="epsilon1_spinbox">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>-12</number>
                </property>
                <property name="maximum">
                 <number>-3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="epsilon2_label">
                <property name="text">
                 <string>Epsilon2 = 10^</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="epsilon2_spinbox">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>-12</number>
                </property>
                <property name="maximum">
                 <number>-3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="l_label">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>L</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="l_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>10</string>
                </property>
                <property name="placeholderText">
                 <string>100</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="min_kier_label">
              <property name="text">
               <string>Direction min. alg. param.</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="tau_label">
                <property name="text">
                 <string>tau_r</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="tau_spinbox">
                <property name="maximumSize">
                 <size>
                  <width>55</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="beta_label">
                <property name="text">
                 <string>beta</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="beta_spinbox">
                <property name="maximumSize">
                 <size>
                  <width>55</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="epsilon2_label_2">
                <property name="text">
                 <string>epsilon</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="epsilon_spinbox">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>-12</number>
                </property>
                <property name="maximum">
                 <number>-3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="oblicz">
          <property name="text">
           <string>Optimize</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="Wykres" name="wykres" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="min_kier_label_2">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>x1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="x_1l_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>-4</string>
            </property>
            <property name="placeholderText">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="x_1p_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
            <property name="placeholderText">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="min_kier_label_3">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>x2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="x_2l_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>-4</string>
            </property>
            <property name="placeholderText">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="x_2p_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>4</string>
            </property>
            <property name="placeholderText">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="plot">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="wyniki">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>tau_label</zorder>
   <zorder>beta_label</zorder>
   <zorder>beta_spinbox</zorder>
   <zorder>tau_spinbox</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>epsilon2_label_2</zorder>
   <zorder>min_kier_label</zorder>
   <zorder>funkcja_celu_box</zorder>
   <zorder>x_1l_edit</zorder>
   <zorder>x_1p_edit</zorder>
   <zorder>x_2p_edit</zorder>
   <zorder>x_2l_edit</zorder>
   <zorder>min_kier_label_2</zorder>
   <zorder>min_kier_label_3</zorder>
   <zorder>plot</zorder>
   <zorder>horizontalSpacer_2</zorder>
   <zorder>horizontalSpacer_3</zorder>
   <zorder>horizontalSpacer_4</zorder>
   <zorder>verticalSpacer_3</zorder>
   <zorder>verticalSpacer_4</zorder>
   <zorder>horizontalSpacer_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Wykres</class>
   <extends>QWidget</extends>
   <header>wykres.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
